forge-gui/res/cardsfolder/upcoming/agency_outfitter.txt

```diff
@@ -0,0 +1,10 @@
+Name:Agency Outfitter
+ManaCost:4 U U 
+Types:Creature Sphinx Detective
+PT:4/3
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSearch | OptionalDecider$ You | TriggerDescription$ When CARDNAME enters the battlefield, you may search your graveyard, hand, and/or library for a card named Magnifying Glass and/or a card named Thinking Cap and put them onto the battlefield. If you search your library this way, shuffle.
+SVar:TrigSearch:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | DifferentNames$ True | ChangeType$ Card.namedMagnifying Glass,Card.namedThinking Cap | ChangeNum$ 2 | ShuffleNonMandatory$ True
```

Comment:
needs all zones handled correctly

Reply:
Seems to work now in testing ?

Reply:
how will you find one from your hand + another in your library?

Reply:
Any pointers ?

Reply:
hmn if that works it's ok

Reply:
From my testing it seems to work 

![image](https://github.com/Card-Forge/forge/assets/67333662/fcb0cd46-9f9d-450e-a9cf-988fab537f79)